src/telemetry/generated/appEvents.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
+
+import EventMetricType from "@mozilla/glean/private/metrics/event";
```

Comment:
Please let's not commit generated code and, in fact, let's add the generated dir to the .gitignore to be sure :)